contracts/yieldStrategy/CurveYieldStrategy.sol

```diff
@@ -161,11 +165,15 @@ contract CurveYieldStrategy is EightyTwentyRangeStrategyVault {
 
     /// @notice claims the accumulated CRV rewards from the gauge, sells CRV rewards for LP tokens and stakes LP tokens
     function _harvestFees() internal override {
-        uint256 claimable = gauge.claimable_reward(address(this), address(crvToken)) + crvPendingToSwap;
+        uint256 before = crvToken.balanceOf(address(this));
+        IGaugeFactory(0xabC000d88f23Bb45525E447528DBF656A9D55bf5).mint(address(gauge));
```

Comment:
Should we move this `IGaugeFactory` to the top as a constant? It should not break storage layout, as well as reduce inline hardcoding.